proto/reaction.proto

```diff
@@ -662,7 +791,9 @@ message Selectivity {
     CUSTOM = 1;
     EE = 2;  // Enantiomeric excess as a percentage.
     ER = 3;  // Enantiomeric ratio. (x:1)
-    DE = 4;  // Diasteromeric ratio (x:1)
+    DR = 4;  // Diasteromeric ratio. (x:1)
+    EZ = 5;  // Alkene geometry ratio. (x:1)
+    ZE = 6;  // Alkene geometry ratio. (x:1)
```

Comment:
Great addition. Maybe in the comments for these selectivity types, we should specify that `EE`, `ER`, and `DR` follow the convention that any product labeled as desired is the one we treat as desired (duh), but `EZ` and `ZE` follow an absolute definition where `EZ` is always the ratio of trans to cis. Either `EZ` or `ZE` can be specified.

Reply:
Agreed. I added definitions to the comments. Feel free to review and change as you see fit.